### ISSUE TITLE
user-requested bug-report

### ISSUE THREAD

**Issue by jjconstrgh** [ubports/ubports-installer#1200]
*Automatically generated error report* 
UBports Installer Version: 0.4.18-beta 
Device: hammerhead
OS to install: Ubuntu Touch
Settings: {"channel":"ubports-touch/16.04/stable","wipe":false,"bootstrap":true,"format":false}
Package: snap
Operating System: Ubuntu Linux 18.04 bionic x64 
NodeJS version: v12.0.0 

Error log: https://paste.ubuntu.com//p/RYXV4yXgfb/

**Comment by jjconstrgh** [comment]
I replaced the power switch on a Nexus 5 in boot loop, and afterward it still had a boot loop, but was able to access UBports installer, unlock OEM, but after that, the boot loader would not stay on. By that I mean a couple times I saw the screen of arrows coming down. Other times, the installer would say 'waiting for device'. After a while. it would say 'yikes', error. Selecting 'recovery' did not fix the interrupted install as it did in my first two Nexus 5's. The phone displayed a new, smaller icon and the words, 'No Command'.

When 'waiting for device' timed out, the phone no longer was OEM unlocked. Repeating the installation process several times resulted in an error message;

Error:fastboot:flash:error:flashing failed:error:{"error":{"killed" :false,"code":1,"signal":null,:"cmd":/snap/ubports-installer/275/app/resources/app.aser.unpacked/platform-tools/linux/fastboot flash recovery "/home/jjconstr/snap/ubports-installer/275/.cache/ubports/hammerhead/firmware/recovery-hammerhead.img""},"stdout":"",stderr":"Sending 'recovery' (11828 KB) OKAY [ 0.740s]\nWriting 'recovery' FAILED (remote: 'flash write failure')\nfastboot: error: Command failed"}

Thanks.
